docker-hub/billing/index.md

```diff
@@ -15,7 +15,7 @@ Docker offers pricing plans that are tailored for individual developers and deve
 
 **For individuals:**
 
-The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, and 2 parallel builds, starting at $5 per month with the annual subscription.
+The **Pro** plan includes unlimited public repositories, unlimited private repositories, unlimited [collaborators](../repos.md#collaborators-and-their-role) for public repositories, one [service account](../repos.md#service-accounts) for private repositories, autobuilds, and 5 parallel builds, starting at $5 per month with the annual subscription.
```

Comment:
perhaps we can reword it as ` ... , auto builds with 5 parallel builds, starting at ....`

Reply:
Thank you! @shaunmulligan Updated